ledger/participant-state/kvutils/src/main/scala/com/daml/ledger/validator/batch/ConflictDetection.scala

```diff
@@ -99,7 +101,8 @@ class ConflictDetection(val damlMetrics: Metrics) {
   // Attempt to produce a useful message by collecting the first conflicting
   // contract id or contract key.
   private def explainConflict(conflictingKeys: Iterable[DamlStateKey]): String =
-    conflictingKeys.toStream
+    conflictingKeys
+      .to(LazyList)
       .sortBy(_.toByteString.asReadOnlyByteBuffer())
```

Comment:
Doesn't make sense to fix in this PR, but sorting a ~stream~ lazy list smells a bit.

Reply:
good point, I’ll leave that to someone who actually understands kvutils :slightly_smiling_face:

Reply:
fixed in af35d721baa0b5e440089c0b1bcd5c5d3c5915f2